features/FEATURE_BLE/ble/pal/PalGap.h

```diff
@@ -44,6 +44,23 @@ struct Gap {
             ble::phy_t rx_phy
         ) = 0;
 
+        /**
+         * @copydoc Gap::EventHandler::onAttMtuChanged
+         */
+        virtual void on_att_mtu_changed(
```

Comment:
How does it defer from `AttExchangeMTUResponse` generated by the pal gatt client ?